common/eth2_testnet_config/build.rs

```diff
@@ -1,37 +1,53 @@
 //! Downloads a testnet configuration from Github.
 
-use eth2_config::{altona, medalla, Eth2NetDirectory};
+use eth2_config::{altona, medalla, Eth2NetArchiveAndDirectory};
 use handlebars::Handlebars;
 use serde_json::json;
+use std::fs;
 use std::fs::File;
+use std::io;
 use std::io::Write;
+use zip::ZipArchive;
 
-const ETH2_NET_DIRS: &[Eth2NetDirectory<'static>] = &[altona::ETH2_NET_DIR, medalla::ETH2_NET_DIR];
+const ETH2_NET_DIRS: &[Eth2NetArchiveAndDirectory<'static>] =
+    &[altona::ETH2_NET_DIR, medalla::ETH2_NET_DIR];
 
 fn main() {
     for testnet in ETH2_NET_DIRS {
         let testnet_dir = testnet.dir();
+        let archive_fullpath = testnet.archive_fullpath();
+        println!("archive fullpath: {:?}", archive_fullpath);
 
-        if !testnet_dir.exists() {
-            std::fs::create_dir_all(&testnet_dir)
-                .unwrap_or_else(|_| panic!("Unable to create {:?}", testnet_dir));
-
-            match get_all_files(testnet) {
-                Ok(()) => (),
-                Err(e) => {
-                    std::fs::remove_dir_all(&testnet_dir).unwrap_or_else(|_| panic!(
-                        "{}. Failed to remove {:?}, please remove the directory manually because it may contains incomplete testnet data.",
-                        e,
-                        testnet_dir,
-                    ));
-                    panic!(e);
+        if !testnet_dir.exists() && archive_fullpath.exists() {
+            //uncompress archive and continue
+            let archive_file = File::open(&archive_fullpath).unwrap();
+            uncompress(archive_file);
+        }
+    }
+}
+
+fn uncompress(archive_file: File) {
```

Comment:
A general rule of thumb is to not use `unwrap()` in any working code in lighthouse (except for tests). We should always handle the errors and only panic as an absolute last resort. If we do have to panic, we should give a helpful description as to why. 

For this function, you can handle the errors more ergonomically by letting it return a result such as `Result<(), io::Error>`. Depending on the error types being produces, maybe you want to return a String here and print it in an explicit panic if it fails. 
The errors in the function can then either be mapped to a String via `map_err()` or potentially already use `io::Error`. Then you can replace the unwraps with `?`.